### ISSUE TITLE
Broken dashboard after error file yaml corruption

### ISSUE THREAD

**Issue by rrelmy** [riasvdv/statamic-redirect#21]
Not sure how it happend

```
[2021-09-01 14:37:28] production.ERROR: Unable to parse at line 182 (near "TML, like Gecko) Chrome/92.0.4515.159 Safari/537.36 Edg/92.0.902.84'"). {"userId":"5793e26f-6336-45f0-8113-8d83e127da94","exception":"[object] (Statamic\\Yaml\\ParseException(code: 0): Unable to parse at line 182 (near \"TML, like Gecko) Chrome/92.0.4515.159 Safari/537.36 Edg/92.0.902.84'\"). at /.../storage/redirect/errors/b7b041b9-09fd-4b0c-a06c-80ce900d780b.yaml:182)
```

In the error yaml it looks like this
``` yaml
handled: true
handledDestination: /fr/distributeur-automatique-de-nourriture-entreprises
TML, like Gecko) Chrome/92.0.4515.159 Safari/537.36 Edg/92.0.902.84'
      ip: xxx.xxx.xxx.xxx
      referer: 'https://www.google.ch/'
```

**Comment by riasvdv** [comment]
Seems like the user agent is breaking the YAML, I'll see if I can fix that.

Best solution is deleting the yaml file for now

**Comment by rrelmy** [comment]
As long as you don't parse/encode YAML manually it looks more like a race condition to me.
Multiple requests writing to the same file.

It did not happen since the report

**Comment by riasvdv** [comment]
Fixed in 1.5.0! The addon now removed the corrupted file and continues working
